test/nightly/getRotatingValue.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+let locationCount: number = getStartingIndex();
+const locations: string[] = ['Australia East', 'Australia Southeast', 'Brazil South', 'Canada Central', 'Central US', 'East Asia', 'East US', 'East US 2', 'France Central', 'Japan East', 'Japan West', 'Korea Central', 'Korea South', 'North Europe', 'South Central US', 'Southeast Asia', 'UK South', 'West Europe', 'West US', 'West US 2'];
+export function getRotatingLocation(): string {
+    locationCount += 1;
+    return locations[locationCount % locations.length];
+}
+
+let pricingTierCount: number = getStartingIndex();
+const pricingTiers: string[] = ['P1v2', 'P2v2', 'P3v2', 'B1', 'B2', 'B3', 'S1', 'S2', 'S3'];
+export function getRotatingPricingTier(): string {
+    pricingTierCount += 1;
+    return pricingTiers[pricingTierCount % pricingTiers.length];
+}
+
+/**
+ * Adds a little more spice to the rotation
+ */
+function getStartingIndex(): number {
```

Comment:
If you need to do `0,5,10` I feel like that's a sign the lists above are too long. We need to strike a balance between testing every possible combination vs making the tests reliable/easy to debug. Right now I think it's too skewed towards the former. Please stick with `0,1,2` and make the lists a bit shorter